pkg/controller/installation/bootstrapReconciler.go

```diff
@@ -158,6 +166,76 @@ func getConsoleRouteCR(ctx context.Context, serverClient k8sclient.Client) (*rou
 	return consoleRouteCR, nil
 }
 
+func (r *Reconciler) reconcilerGithubOauthSecret(ctx context.Context, serverClient k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
+
+	githubOauthSecretCR := &corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      r.ConfigManager.GetGHOauthClientsSecretName(),
+			Namespace: r.ConfigManager.GetOperatorNamespace(),
+		},
+		Data: map[string][]byte{
+			"clientId": []byte("dummy"),
+			"secret":   []byte("dummy"),
+		},
+	}
+
+	err := serverClient.Create(ctx, githubOauthSecretCR)
+	if err != nil {
+		if !k8serr.IsAlreadyExists(err) {
```

Comment:
Very minor, feel free to ignore: this condition can merged with the one the previous line :)